README.md

```diff
@@ -1 +1,44 @@
-# Project1
\ No newline at end of file
+# Project1
+Hello Microverse
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. The goal here is to master all of the tools and best practices I learned . All about gitflow , linters.
+
+set up a new repository and prepare it for development using best practices.
```

Comment:
- [ ] I know that you have worked hard on making the description of this project however your description is explaining the technical tasks of the project rather than a simple overview of the project which may not be necessary to explain your technical task but just a summary of the output of the project. kindly work on that. I know you can do it :clap: :clap: